src/index.js

```diff
@@ -10,9 +10,9 @@ client.on("ready", () => {
     console.log(`Logged in as ${client.user.tag}!`);
 });
 
-client.on("message", msg => {
+client.on("message", msg => {        
     if (msg.content.startsWith("+")) {
-        const message = commandsHandler(msg.content)
+        const message = commandsHandler(msg.content, msg)
```

Comment:
Aqui você passa a `msg` inteira, mas dentro, você só usa o `sender.author.id`, é bom que o caso de uso n˜ão tenha conhecimento do objeto inteiro, somente do valor passado.
O que acha?
```suggestion
        const message = commandsHandler(msg.content, msg.author.id)
```

Reply:
Acho bom, então acho melhor até a gente não passar somente o content, levamos o objeto inteiro la pro command, e la a gente define os objetos q vamos passar para os user cases

Reply:
Pode ser, mas o problema dessa estratégia é que o caso de uso vai precisar ter conhecimento do objeto inteiro, acho que o ideal seria ele trabalhar somente com os parâmetros.
Mas se preferir, pode passar o objeto sim. 😅

Reply:
Olha lá o q eu fiz, eu não levei o objeto inteiro pro caso de uso, eu levei o objeto inteiro pro command, os casos de uso só receberam os valores necessários